pkg/reconciler/pipelinerun/pipelinerun.go

```diff
@@ -80,6 +80,9 @@ const (
 	// ReasonCouldntGetCondition indicates that the reason for the failure status is that the
 	// associated Pipeline's Conditions couldn't all be retrieved
 	ReasonCouldntGetCondition = "CouldntGetCondition"
+	// ReasonCouldntGetParams indicates that the reason for the failure status is that the
+	// associated PipelineRun didn't provide all the required parameters
+	ReasonCouldntGetParams = "CouldntGetParams"
```

Comment:
i would call this something like "MissingParams" - "CouldntGet" is being used above when doing a GET on a resource is failing.

taskruns seem to use 'validation failed' for this case https://github.com/tektoncd/pipeline/blob/58c836d55ef750cd023bae901fec51389a97f318/pkg/reconciler/taskrun/taskrun.go#L300

Above there is a ParameterTypeMismatch reason as well, you could use that as a precedent and make it ParameterMissing